### DISCUSSION TITLE
how to subscribe ?

### DISCUSSION THREAD

**Discussion by montyfert** [arendst/Tasmota#19301] [Q&A]
hi i have a tasmota display and want to subscrive to a topic (octoPrint/plugin/filament_scale/filament_weight) bu i have no idea how to any help would be great. thanks
martin

**Comment by sfromis** [comment]
https://tasmota.github.io/docs/MQTT/#subscribeunsubscribe
https://tasmota.github.io/docs/Berry/#mqtt-module

**Comment by montyfert** [comment]
hi i tried  Subscribe weight, octoPrint/plugin/filament_scale/filament_weight/DisplayText  and get command unknown,

**Reply by sfromis** [reply]
Looks like you "forgot" to read the text in the link I posted about you having to do a build to include the feature.

**Comment by Lumpi-125** [comment]
Do i have to do my own build also if i just want to use the berry mqtt-module?

**Reply by sfromis** [reply]
No, but you need to have a device based on ESP32. In that case, I'd say that you get better features using Berry.

**Comment by montyfert** [comment]
ive compiled the build and (Subscribe weight, octoPrint/plugin/filament_scale/filament_weight/DisplayText) still no luck

**Reply by barbudor** [reply]
'still no luck' doesn't give a hint on what happens
Copy/paste of the console, preferably in Weblog 3 level would help
What about your rule to catch the 'weight' event?

**Reply by montyfert** [reply]
to be honest i havent got a clue what im doing ,
